include/language/en_us.lang.php

```diff
@@ -2400,6 +2403,8 @@
     'LBL_DELETE_BUTTON_KEY' => 'd',
     'LBL_DELETE_BUTTON_LABEL' => 'Delete',
     'LBL_DELETE_BUTTON_TITLE' => 'Delete',
+    'LBL_BULK_ACTION_BUTTON_LABEL' => 'BULK ACTION',
```

Comment:
@horus68 I have updated this PR so that the tabs and the create button are no longer capitalised. I have also changed the labels so they are capitalised. 

If you notice any tabs that are not in capitals (English only), will you please let me know?